<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkPrefWidget</class>
 <widget class="QWidget" name="WkPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo Mode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="enableGroupBox">
     <property name="title">
      <string>Enable Demo Mode</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="runSpeedLabel">
          <property name="text">
           <string>Run Speed Multiplier</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="runSpeedDoubleSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="pauseLabel">
          <property name="text">
           <string>Pause Time on Restart (sec)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="pauseSpinBox">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="selectGroupBox">
        <property name="title">
         <string>Select Platform on Start of Scenario</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="platformLabel">
           <property name="text">
            <string>Platform Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="platformLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>enableGroupBox</tabstop>
  <tabstop>runSpeedDoubleSpinBox</tabstop>
  <tabstop>pauseSpinBox</tabstop>
  <tabstop>selectGroupBox</tabstop>
  <tabstop>platformLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
